cuda/CMakeLists.txt

```diff
@@ -151,9 +151,6 @@ target_include_directories(ginkgo_cuda
     SYSTEM PRIVATE ${CUDA_INCLUDE_DIRS})
 target_link_libraries(ginkgo_cuda PRIVATE ${CUDA_RUNTIME_LIBS} ${CUBLAS} ${CUSPARSE})
 
-# Need to link against ginkgo_hip for the `raw_copy_to(HipExecutor ...)` method
-target_link_libraries(ginkgo_cuda PUBLIC ginkgo_hip)
```

Comment:
I do not know why this was supposed to be needed, maybe at some point I had implemented it as such, but if you look in `hip/base/executor.hip.cpp` there is no definition of `CudaExecutor::raw_copy_to(HipExecutor...)`, which this would imply.